pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -325,12 +325,16 @@ class Build {
                         case "windows": buildWindowsInstaller(versionData); break
                 	default: return; break
                 }
-		try {
-                    context.sh 'for file in $(ls workspace/target/*.tar.gz workspace/target/*.pkg workspace/target/*.msi); do sha256sum "$file" > $file.sha256.txt ; done'
-                    writeMetadata(versionData)
-                    context.archiveArtifacts artifacts: "workspace/target/*"
-                } catch (e) {
-                    context.println("Failed to build ${buildConfig.TARGET_OS} installer ${e}")
+                // Archive the Mac and Windows pkg/msi
+                if (buildConfig.TARGET_OS == "mac" || buildConfig.TARGET_OS == "windows") {
```

Comment:
Can we add a comment explaining why linux isn't in there for people looking at this in the future?